bin/errorHelpers.js

```diff
@@ -4,54 +4,27 @@
 */
 "use strict";
 
-const loaderFlag = "LOADER_EXECUTION";
-
 const webpackOptionsFlag = "WEBPACK_OPTIONS";
 
 exports.cutOffByFlag = (stack, flag) => {
 	stack = stack.split("\n");
 	for (let i = 0; i < stack.length; i++)
-		if (stack[i].indexOf(flag) >= 0) stack.length = i;
+		if (stack[i].indexOf(flag) >= 0)
+			stack.length = i;
 	return stack.join("\n");
 };
 
-exports.cutOffLoaderExecution = stack =>
-	exports.cutOffByFlag(stack, loaderFlag);
-
-exports.cutOffWebpackOptinos = stack =>
-	exports.cutOffByFlag(stack, webpackOptionsFlag);
+exports.cutOffWebpackOptions = (stack) => exports.cutOffByFlag(stack, webpackOptionsFlag);
```

Comment:
Could you tell me what these things are doing?

Reply:
@ev1stensberg the file name change is slightly confused things here. I guess, this was the old errorhelpers content, I think I have to take the latest from the bin/errorhelpers.js right ?

correct me if I am wrong

Reply:
yep, that would be prefered, thought you did that, so that's why I'm asking

Reply:
yup, the old errorhelper was from Tobias, but the current master errorHelper was having all the methods what was there and new methods were added on top of it. So I decided to have this version.

Reply:
So if I understand: You've looked at Tobias's PR, that is where this implementation of ErrorHelper is from?

Reply:
yeah... But when I merged with the master I saw yours  (current master) which has more methods. 

Tobias PR : 
https://github.com/webpack/webpack-cli/pull/251/commits/cf82edb1c35e2f25689d8e293562f2ee501ab761

Reply:
Keep what Tobias refactored, see if there's any old calls using the old code in the CLI, I'll re-review after